content/community/videos.md

```diff
@@ -1,76 +1,83 @@
 ---
 id: videos
-title: Videos
+title: الفيديوهات
 layout: community
 sectionid: community
 permalink: community/videos.html
 redirect_from:
  - "docs/videos.html"
 ---
 
-Videos dedicated to the discussion of React and the React ecosystem.
+مقاطع فيديو مخصصة لمناقشة React ونظامه المتكامل.
 
 ### React.js Conf 2017 {#reactjs-conf-2017}
 
-A playlist of videos from React.js Conf 2017.
+قائمة تشغيل لفيديوهات مؤتمر React.js Conf 2017.
 <iframe title="React.js Conf 2017" width="650" height="366" src="https://www.youtube-nocookie.com/embed/playlist?list=PLb0IAmt7-GS3fZ46IGFirdqKTIxlws7e0" frameborder="0" allowfullscreen></iframe>
 
 ### React.js Conf 2016 {#reactjs-conf-2016}
 
-A playlist of videos from React.js Conf 2016.
+قائمة تشغيل لفيديوهات مؤتمر React.js Conf 2016.
 <iframe title="React.js Conf 2016" width="650" height="366" src="https://www.youtube-nocookie.com/embed/playlist?list=PLb0IAmt7-GS0M8Q95RIc2lOM6nc77q1IY" frameborder="0" allowfullscreen></iframe>
 
 ### React.js Conf 2015 {#reactjs-conf-2015}
 
-A playlist of videos from React.js Conf 2015.
+قائمة تشغيل لفيديوهات مؤتمر React.js Conf 2015.
 <iframe title="React.js Conf 2015" width="650" height="366" src="https://www.youtube-nocookie.com/embed/playlist?list=PLb0IAmt7-GS1cbw4qonlQztYV1TAW0sCr" frameborder="0" allowfullscreen></iframe>
 
 ### Secrets of the Virtual DOM {#secrets-of-the-virtual-dom}
 
-Pete Hunt at Mountain West JavaScript 2014 discusses why a virtual DOM was built for React, how it compares to other systems, and its relevance to the future of browser technologies - (2014 - 0h44m).
+يناقش بيت هانت في  ماونتن ويست جافا سكريبت 2014 سبب إنشاء DOM افتراضي لـ React ، وكيفية مقارنته بالأنظمة الأخرى ، وأهميته بالنسبة لمستقبل تقنيات المتصفح.
+<!--  - (2014 - 0h44m) -->
```

Comment:
> Hi Imed thanks again for your efforts, I left below some comments and suggestions. Also please uncomment and translate the video lengths like in the line 31.
> Also, what do you think of translating titles that doesn't have the conference name or something, like those who have simple sentences like `React and Flux: Building Applications with a Unidirectional Data Flow`, `Introduction to React` ?

I have put "duration" in the comment tag because it has a big effect on description formatting ( there are no words after it - rtl pb)  ...

I think in terms of coordination all the titles should be translated not each other ..

Reply:
We can find a workaround later, let's leave it like this, for now. Commenting it was a good idea :)

Reply:
ok bro , nchlh 💯